utils/template.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"bytes"
+	"html/template"
+)
+
+// Merge merges data into the template tpl and returns the result.
+func Merge(tpl []byte, data interface{}) ([]byte, error) {
```

Comment:
@mgfeller should this be called `MergeToTemplate()`? Because utils package is not specific to templating use case.

Reply:
Sure. I'll change it.